subprojects/docs/src/samples/testing/jacoco/quickstart/kotlin/build.gradle.kts

```
@@ -65,3 +65,23 @@ tasks.jacocoTestCoverageVerification {
     }
 }
 // end::violation-rules-configuration[]
+
+// tag::testtask-configuration-defaults[]
+tasks.getByName<Test>("test") {
+    extensions.configure(JacocoTaskExtension::class) {
+        isEnabled = true
+        destinationFile = file("$buildDir/jacoco/$name.exec")
+        includes = listOf()
```

:o: using `emptyList()` might be clearer, applies to other list properties below
```suggestion
        includes = emptyList()
```